osu.Game/Overlays/Mods/ModMapInfoContainer.cs

```diff
@@ -0,0 +1,184 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+#nullable disable
+
+using osu.Framework.Allocation;
+using osu.Framework.Bindables;
+using osu.Framework.Extensions.LocalisationExtensions;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Colour;
+using osu.Framework.Graphics.Shapes;
+using osu.Framework.Localisation;
+using osu.Game.Beatmaps;
+using osu.Game.Beatmaps.Drawables;
+using osu.Game.Graphics;
+using osu.Game.Graphics.Sprites;
+using osu.Game.Graphics.UserInterface;
+using osuTK;
+using osuTK.Graphics;
+
+namespace osu.Game.Overlays.Mods
+{
+    public partial class ModMapInfoContainer : Container
+    {
+        private Container content;
+        private Container innerContent;
+
+        private Box background;
+        private Box innerBackground;
+
+        private StarRatingDisplay starRatingDisplay;
+        private BPMDisplay bpmDisplay;
+
+        private VerticalAttributeDisplay circleSizeDisplay;
+        private VerticalAttributeDisplay drainRateDisplay;
+        private VerticalAttributeDisplay approachRateDisplay;
+        private VerticalAttributeDisplay overallDifficultyDisplay;
+
+        [Resolved]
+        private OverlayColourProvider colourProvider { get; set; }
+
+        [Resolved]
+        private Bindable<BeatmapShortInfo> adjustedInfo { get; set; }
+
+        public ModMapInfoContainer()
+        {
+            // values as ModSelectOverlay footer buttons
+            const float shear = ShearedOverlayContainer.SHEAR;
+            const float corner_radius = 7;
+            const float border_thickness = 2;
+
+            AutoSizeAxes = Axes.Both;
+            InternalChild = content = new InputBlockingContainer
+            {
+                Origin = Anchor.BottomRight,
+                Anchor = Anchor.BottomRight,
+                AutoSizeAxes = Axes.X,
+                Height = 50, // as ModSelectOverlay footer buttons
```

Comment:
Extract `ShearedButton.Height` to a constant (`public const double HEIGHT = 50;`) and reference that instead.

Reply:
fixed